scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -634,7 +634,7 @@ trait CliTestBehavior { this: AbstractCliTest =>
         assertOut = out => {
           assertContains(
             out,
-            """|foo.scala-formatted
+            """|foo.scala
```

Comment:
should we check for `b/` as well?

Reply:
Checking for /b does makes sense since it is represents foo.scala-formatted, will get to it